### PR TITLE
Add support for TS.QUERYINDEX TS.ALTER TS.INCRBY/TS.DECRBY TS.MADD

### PR DESCRIPTION
**Pull request by dengliming**
for [https://github.com/RedisTimeSeries/redistimeseries-go/issues/53](https://github.com/RedisTimeSeries/redistimeseries-go/issues/53)、[https://github.com/RedisTimeSeries/redistimeseries-go/issues/57](https://github.com/RedisTimeSeries/redistimeseries-go/issues/57)、[https://github.com/RedisTimeSeries/redistimeseries-go/issues/56](https://github.com/RedisTimeSeries/redistimeseries-go/issues/56)、[https://github.com/RedisTimeSeries/redistimeseries-go/issues/55](https://github.com/RedisTimeSeries/redistimeseries-go/issues/55)、[https://github.com/RedisTimeSeries/redistimeseries-go/issues/54](https://github.com/RedisTimeSeries/redistimeseries-go/issues/54)

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -275,3 +288,65 @@ func (client *Client) Info(key string) (res KeyInfo, err error) {
...
+	args := redis.Args{key, value}
+	if timestamp > 0 {
+		args = args.Add("TIMESTAMP", timestamp)
+	}
+
+	return options.Serialize(args)
+}
+
+// Append new samples to a list of series.
+func (client *Client) MultiAdd(samples ...Sample) ([]interface{}, error) {
```
Comment (filipecosta90) [inline client.go:343]: hi there @dengliming the TS.MADD accepts multiple keys as input. Your approach is correct, but does not allow for samples from different series.
example:
```
127.0.0.1:6379>TS.MADD temperature:2:32 1548149180000 26 cpu:2:32 1548149183000 54
```
happy to approve the other changes while we work on MADD in another PR, or to fix this one :) you decide what suits you best.

Comment (filipecosta90) [inline client.go:343]: hi there @dengliming the TS.MADD accepts multiple keys as input. Your approach is correct, but does not allow for samples from different series.
example:
```
127.0.0.1:6379>TS.MADD temperature:2:32 1548149180000 26 cpu:2:32 1548149183000 54
```
happy to approve the other changes while we work on MADD in another PR, or to fix this one :) you decide what suits you best.

Comment (dengliming) [reply client.go:343]: I'm not sure I understand what you mean. Now the `Sample `struct includes `Key `field. That should be work for samples from different series.
```
type Sample struct {
	Key       string
	DataPoint DataPoint
}
```

Comment (filipecosta90) [reply client.go:343]: you're totally right @dengliming. Sorry for the confusion. My mistake =)

Comment (filipecosta90) [reply client.go:343]: no real issue. all good here

**Comment by dengliming** [comment]
It seems that CI/TEST has been taking a long time.

**Comment by filipecosta90** [comment]
> It seems that CI/TEST has been taking a long time.

for some reason the circle build on PRs was disabled. I believe the fastest way to trigger it properly is to add another test to force a code change. I don't see a way on circle to trigger the build myself. Can we add one more test ( for example on Madd with multiple series )? That should do it =)

**Comment by filipecosta90** [comment]
thank you for contribution @dengliming =)

**Pull request merged**